ui/component/or-attribute-card/src/index.ts

```diff
@@ -0,0 +1,441 @@
+import {css, customElement, html, LitElement, property, PropertyValues, query, unsafeCSS} from "lit-element";
+import {classMap} from "lit-html/directives/class-map";
+import i18next from "i18next";
+import {Asset, DatapointInterval, MetaItemType, ValueDatapoint} from "@openremote/model";
+import {manager, DefaultColor4, DefaultColor5} from "@openremote/core";
+import Chart, {ChartTooltipCallback} from "chart.js";
+import {getContentWithMenuTemplate} from "@openremote/or-chart";
+import {InputType, OrInputChangedEvent} from "@openremote/or-input";
+import {OrAttributeHistoryEvent} from "@openremote/or-attribute-history";
+import {getMetaValue} from "@openremote/core/dist/util";
```

Comment:
Make sure or-package references are correctly listed in `tsconfig.json` and `package.json`